jgrapht-core/src/main/java/org/jgrapht/traverse/LexBreadthFirstIterator.java

```diff
@@ -33,7 +34,7 @@
  * Iterator chooses vertex with lexicographically largest label and returns it. It breaks ties
  * arbitrarily. For more information on lexicographical BFS see the following article: Corneil D.G.
  * (2004) <a href="https://pdfs.semanticscholar.org/d4b5/a492f781f23a30773841ec79c46d2ec2eb9c.pdf">
- * <i>Lexicographic Breadth First Search – A Survey</i></a>. In: Hromkovič J., Nagl M., Westfechtel
+ * <i>Lexicographic Breadth First Search – A Survey</i></a>. In: Hromkovic J., Nagl M., Westfechtel
```

Comment:
dunno if it is nice to write their name wrong (although it might by better for encoding)

Reply:
Actually, this is why I changed it -- it would not build with the original name.